server/jetstream_api.go

```diff
@@ -1596,6 +1680,28 @@ func (s *Server) sendJetStreamAPIAuditAdvisory(c *client, subject, request, resp
 	}
 }
 
+func (c *client) apiAuditClient() ClientAPIAudit {
+	c.mu.Lock()
```

Comment:
Can do defer on unlock and do direct assignment. I think defer here is ok.

Reply:
`c.Account()` again locks and i didnt want to mess about with that, so copied old code pattern